cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
What do you mean by 100 solution? I agree that one more field to fill in is bad, but there is nothing we can do to set up well name that will be clear.